packages/botonic-plugin-contentful/src/nlp/normalizer.ts

```diff
@@ -86,25 +105,25 @@ export class Normalizer {
     const stemmer = stemmerFor(locale)
     // tokenizer will replace i18n characters
     const tokens = this.tokenizer(locale).tokenize(txt, true)
-    let stems: string[] = []
+    let words: Word[] = []
     const stopWords = this.stopWordsPerLocale[locale]
+    let numStopWords = 0
     for (const token of tokens) {
       const black = this.getBlackListStem(locale, token)
```

Comment:
`if isBlacklist`...

Reply:
actually it needs to be like this, because getBlackListStem returns the stem from the blacklist structure.
Eg. getBlackListStem("pedidos") will return "pedido)